### DISCUSSION TITLE
개발자님 질문이용

### DISCUSSION THREAD

**Discussion by ranware** [changdoc/naver-vaccine-macro#249] [Q&A]
카카오맵에서 할때 카카오 로그인을 꼭 해야되나요?

**Comment by changdoc** [comment]
예약 시도할 백신 종류를 선택하고 동작 시킨다면 카카오 로그인을 꼭 해야합니다.

백신 종류 선택을 하고 동작하게 될때 백신 종류별로 잔여 백신 수를 조회 하게되는데요. 카카오 로그인이 안되어 있으면 종류별 백신 수량 체크를 할 수 없게됩니다.

**Reply by ranware** [reply]
아하 감사합니다
